content/battery.md

```diff
@@ -19,167 +19,155 @@ section: hardware
 
-```bash
-sudo gedit /etc/tlp.conf
-```
-
-The `/etc/tlp.conf` file is for user configuration while `/etc/tlp.d/*.conf` file is for drop-in customization snippets.
-
-All of the info about the program can be found with these 2 commands:
-
-```bash
-man tlp
-man tlp-stat
-```
+![disable-wifi](/images/battery/disable-wifi.png)
 
----
+## Monitoring system performance
 
-### Powertop
+Pop!\_OS has a number of different options for monitoring the CPU and GPU use of processes and applications. The more CPU and/or GPU a program is using, the more battery life it will consume as well. If a system's battery isn't lasting as long as expected, it can be useful to see what the system is doing to go through the charge more quickly.
 
-Powertop monitors running processes and applications to determine how much power is being consumed. Powertop can also use application battery usage data to create custom power profiles. We recommend installing and running TLP before using Powertop.
+### `powertop`
 
-To install <u>powertop</u>, please open a terminal and run this command:
+`powertop`is a tool for generating reports about what applications and hardware are using the most power, as well as providing live monitoring of the system. It can be installed by running this command in the terminal:
 
 ```bash
 sudo apt install powertop
 ```
 
-After installing the program, reboot your computer and calibrate the readings on battery power with this command:
+And the following command will generate a report:
 
 ```bash
-sudo powertop -c
+sudo powertop --html=report.html
 ```
 
-This will take about 15 minutes to run the calibration. The system will turn the display off a few times, and you won't be able to do anything else on the PC during the process.  Powertop can be run just by itself to see what is using resources on your system.  It needs to be left open for a little amount of time to gather statistics, and be more accurate.  Run it with this command:
+The `report.html` file can be opened in a web browser and will outline what processes and hardware used the most power during its testing. It also has recommended settings on the "Tuning" page, which may or may not have undesireable side effects; the commands on the "Tuning" page to change those settings are temporary and will be cleared if the system is rebooted. Caution is suggested when trying these suggestions.
 
-```bash
-powertop
-```
-
-Powertop can also generate HTML reports with this command:
+Running the command in the terminal without any parameters like this:
 
 ```bash
-sudo powertop --html=report.html
+sudo powertop
 ```
 
-Open the report located at `~/report.html` to see the results.
+will start `powertop` in an interactive monitoring mode. The Tab key can be used to move between pages and the arrow keys to scroll, and it's very similar to to HTML report, only it's updated in real-time.
```

Comment:
`<kbd>` tags for the Tab key.

> The Tab key can be used to move between pages and the arrow keys to scroll, and it's very similar to to HTML report, only it's updated in real-time.

I would break this into two sentences. Possibly: Use the <kbd>Tab</kbd> key to navigate between pages, and the arrow keys to scroll. This output shows the same information from the HTML report, but is updated in real-time.

Reply:
Done, used your suggested wording. Will be updated on the next git push.